signing/secp256k1/impl/build.gradle

```diff
@@ -25,26 +26,36 @@ jar {
   }
 }
 
+testFixturesJar {
+  archiveBaseName = calculateJarName(project)
```

Comment:
does this create the right named testfixture jar?

Reply:
I copied this from what the hashicorp module was doing so it should be. I'll try and confirm. It is working correctly if I publish to maven locally